cernopendata/modules/fixtures/data/records/cms-tools-analysis-examples.json

```diff
@@ -0,0 +1,96 @@
+[
+{
+  "abstract": {
+    "description": " <p>This research level example is a strongly simplified reimplementation of parts of the original CMS Higgs to four lepton analysis published in <a href=\"https://inspirehep.net/record/1124338?ln=en\">Phys.Lett. B716 (2012) 30-61,  arXiv:1207.7235</a>.</p> <p>The published reference plot which is being approximated in this example is <a href=\"https://inspirehep.net/record/1124338/files/H4l_mass_v3.png>https://inspirehep.net/record/1124338/files/H4l_mass_3.png</a>. Other Higgs final states (e.g. Higgs to two photons), which were also part of the same CMS paper and strongly contributed to the Higgs boson discovery, are not covered by this example.</p> <p>The example addresses users who feel they have at least some minimal understanding of the content of this paper and of the meaning of this reference plot.It requires a minimal acquaintance with the linux operating system and the ROOT analysis package (<a href=\"https://root.cern.ch/\">https://root.cern.ch/</a>).</p> "
```

Comment:
In the abstract, the portion `mass_v3.png>https://inspirehep.ne` is missing the ending quote, should be: `mass_v3.png\">https://inspirehep.ne`